src/dymaptic.GeoBlazor.Core/DependencyExtension.cs

```diff
@@ -0,0 +1,21 @@
+﻿using dymaptic.GeoBlazor.Core.Model;
+using Microsoft.Extensions.DependencyInjection;
+
+
+namespace dymaptic.GeoBlazor.Core;
+
+/// <summary>
+///     Static extension class for injecting GeoBlazor types
+/// </summary>
+public static class DependencyExtension
+{
+    /// <summary>
+    ///     Adds the Logic components <see cref="GeometryEngine"/> and <see cref="Projection"/> to your dependency
+    ///     injection collection.
+    /// </summary>
+    public static IServiceCollection AddGeoBlazor(this IServiceCollection serviceCollection)
+    {
+        return serviceCollection.AddScoped<GeometryEngine>()
```

Comment:
Is scoped the right level here? These are static in the JS api.

Reply:
Good point, I think I'll change these to Singletons to match

Reply:
Testing on develop (should have tested before merging), I have to put these back. `JSRuntime` is scoped, so these have to be as well, or it throws an exception. I think that's fine, however, because the scope encompasses a page and internal navigation until a user or code refreshes the browser.